src/vm/finalizerthread.cpp

```diff
@@ -20,6 +20,12 @@
 BOOL FinalizerThread::fRunFinalizersOnUnload = FALSE;
 BOOL FinalizerThread::fQuitFinalizer = FALSE;
 
+#if defined(FEATURE_PAL)
+#include "unixlowmem.h"
+
+#define LOWMEMORY_CHECK_TIMEOUT 100
```

Comment:
Does this mean we are going to have the finalizer thread waking up every 100ms even when everything is idle?